libraries/config/src/network.cpp

```diff
@@ -4,6 +4,16 @@
 
 namespace taraxa {
 
+void dec_json(const Json::Value &json, PrometheusConfig &config) {
+  if (auto listen_port = getConfigData(json, {"listen_port"}, true); !listen_port.isNull()) {
```

Comment:
if whole section "prometheus" is optional, then the values in it dont should not be optional I guess - if you specify "primetheus", you must specify also things in it